GovUk.Frontend.Umbraco/Models/TprHeaderLockupModel.cs

```diff
@@ -16,6 +16,10 @@ public TprHeaderLockupModel(IPublishedContent settings)
         public bool ShowSkipLink { get; init; } = true;
         public bool ShowPhaseBanner { get; init; } = true;
         public bool ShowHeader { get; init; } = true;
+        public bool HeaderBarContentAllowHtml { get; init; } = true;
```

Comment:
The `init` keyword means that a value can only be assigned during construction. That makes it immutable after construction. How does that make it configurable? It can't be changed.